projects/cilium/cilium/README.md

```diff
@@ -15,4 +15,4 @@ The Cilium operator image includes the generic cilium-operator binary and the `g
 2. Install `skopeo` if you don't have it ([instructions](https://github.com/containers/skopeo/blob/main/install.md)).
 3. Run `make update-digests` in this folder.
 4. Update the version at the top of this Readme.
-5. Run `make generate` from the root of the repo to update the UPSTREAM_PROJECTS.yaml file.
\ No newline at end of file
+5. Run `make generate` from the root of the repo to update the UPSTREAM_PROJECTS.yaml file.
```

Comment:
Just curious why did github show that this line was changed

Reply:
No idea actually, maybe the way that the file saved or something

Reply:
When the file was saved, a newline got added to it by the IDE. That's the diff you're seeing here.